x-pack/plugins/maps/common/constants.ts

```diff
@@ -334,3 +334,7 @@ export enum WIZARD_ID {
   WMS_LAYER = 'wmsLayer',
   TMS_LAYER = 'tmsLayer',
 }
+
+// Maplibre does not provide any feedback when rendering is complete.
+// Workaround is hard-coded timeout period.
+export const RENDER_TIMEOUT = 1000;
```

Comment:
Reduced timeout from 5 seconds to 1 seconds because when timeout was first added, EMS basemap layers did not have great loading status. Now they do so this timeout seemed excessive.